apps/public/src/app/(map)/locatie/kies/_components/ListView/ListView.module.css

```diff
@@ -0,0 +1,16 @@
+.container {
+  background-color: white;
+  height: 100%;
+  position: absolute;
```

Comment:
Why is this positioned absolute?

Reply:
Its positioned on top of the map.

Reply:
It is, but `position: absolute` doesn't really have anything to do with that. The default is `position: static`, which means the `z-index` doesn't work. We should probably change this to `position: relative`